src/course-outline/CourseOutline.jsx

```diff
@@ -235,7 +235,7 @@ const CourseOutline = ({ courseId }) => {
                           </DraggableList>
                           <Button
                             data-testid="new-section-button"
-                            className="mt-4"
+                            className="mt-4 bg-light-100 border-gray-200"
```

Comment:
Remove the background and border color. The hover state when these are changed cause an accessibility contrast error. The default styling for the `outline-primary` variant matches the design in figma. This also applies to the new subsection and new unit buttons.

Reply:
@KristinAoki The designs show white background for new section and new unit button but the default styling for `outline-primary` variant has transparent background so it becomes grayish.

Reply:
That is ok, because it is the expected behavior for the open edx button styles. The mocks are built using the edx styles so not everything is going to match if you use the default open edx styles.

Reply:
@KristinAoki Got it. Updated.